JAS0NHUANG/ft_transcendence
# Api

To test:
- signup non-42 user: @Post request http://localhost:3000/auth/local/signup in Postman/Insomnia with body x-www form {userName, hash, email}. A cookie is then created.
- login non-42 user: @Post request http://localhost:3000/auth/local/login in Postman/Insomnia with body x-www form {hash, email}. 
- update: @Patch request http://localhost:3000/user/me/update, make sure the right jwt token is in the cookie when sending the request, check changes in prisma studio
